test/icinga-legacytimeperiod.cpp

```diff
@@ -691,4 +691,56 @@ BOOST_AUTO_TEST_CASE(dst_isinside)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(find_nth_weekday) {
+	auto run = [](const std::string &refDay, int wday, int n, const std::string &expectedDay) {
```

Comment:
For the sake of consistency, the`&` of the lambda arguments should be left-aligned, otherwise looks fine for me.